manifests/config.pp

```diff
@@ -61,5 +61,12 @@
     group   => 'root',
     content => $::openvpn::dh,
   }
+  file { '/etc/openvpn/keys/ta.key':
+    ensure  => present,
+    mode    => '0600',
+    owner   => 'root',
+    group   => 'root',
+    content => $::openvpn::ta,
```

Comment:
dh isnt mandatory it can be undef too - according to the manifest/init.pp I believe.